ci/docker/Dockerfile-nighthawk-benchmark

```diff
@@ -0,0 +1,24 @@
+FROM frolvlad/alpine-glibc
+
+CMD ["./benchmarks" "--help"]
```

Comment:
Forgive me, I've been out of docker for a while.  Why is it necessary for us to run --help at the start of this?

Reply:
I took this verbatim from the `docker history` of the original container.  I want to think that this is a quick way to ensure that all the python dependencies for `benchmarks` exist.  If not, the container build would fail immediately.